### ISSUE TITLE
Change `components` settings?

### ISSUE THREAD

**Issue by dv** [foundation/inky-rb#52]
The README talks of **Custom Elements** and how the names of tags can be changed with the `components` settings, but there does not actually seem to be a way to change it. The `options` param in `Inky::Core` seems to be the way to go, but in `Rails` there's no way to pass in any value into that param as far as I can tell.

**Comment by marcandre** [comment]
My opinion is that this is a misfeature. I can't quite see a good usecase for this feature (in Inky or Inky-rb).

**Comment by dv** [comment]
Makes sense yeah. Should probably be removed from the README then to avoid confusion.

**Comment by joeworkman** [comment]
This repository is being sunset. The Inky project is being rewritten in Rust with a single canonical core and official bindings for all major languages — including a new Ruby gem that uses the shared Rust core via FFI (fiddle).

This means:
- One implementation, identical output across all languages
- No more Ruby-specific bugs or maintenance burden
- The new Ruby gem will have zero gem dependencies (fiddle is in Ruby's stdlib)

You can follow progress at https://github.com/foundation/inky.

Thanks for using inky-rb!
